lib/logstash-core.rb

```diff
@@ -0,0 +1,2 @@
+module LogStash
```

Comment:
Are we good with changing the name of the main file to logstash-core and getting rid of logstash ? I might make sense to change if we understand the code as a self contained gem